modules/nw-sriov-network-attachment.adoc

```diff
@@ -28,20 +28,35 @@ apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetwork
 metadata:
   name: <name> <1>
-  namespace: sriov-network-operator <2>
+  namespace: openshift-sriov-network-operator <2>
 spec:
   networkNamespace: <target_namespace> <3>
   ipam: |- <4>
     ...
   vlan: <vlan> <5>
   resourceName: <sriov_resource_name> <6>
+  linkState: <link_state> <7>
+  maxTxRate: <max_tx_rate> <8>
+  minTxRate: <mix_rx_rate> <9>
+  vlanQoS: <vlan_qos> <10>
+  spoofChk: <spoof_check> <11>
+  trust: <trust_vf> <12>
+  capabilities: <capabilities> <13>
+  
 ----
 <1> Replace `<name>` with a name for the CR. The Operator will create a NetworkAttachmentDefinition CR with same name.
 <2> Specify the namespace where the SR-IOV Operator is installed.
-<3> Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR will be created.
-<4> Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
-<5> Replace `<vlan>` with a Virtual LAN (VLAN) ID for the additional network. The integer value must be from `0` to `4095`. The default value is `0`.
+<3> Optional. Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR will be created.
+<4> Optional. Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<5> Optional. Replace `<vlan>` with a Virtual LAN (VLAN) ID for the additional network. The integer value must be from `0` to `4095`. The default value is `0`.
 <6> Replace `<sriov_resource_name>` with the value for the `.spec.resourceName` parameter from the SriovNetworkNodePolicy CR that defines the SR-IOV hardware for this additional network.
+<7> Optional. Replace `<link_state>` with a the link state of VF. Allowed value are `enable`, `disable` and `auto`. The default value is `auto`.
+<8> Optional. Replace `<max_tx_rate>` with a Maximum tx rate, in Mbps, for the VF. The default value is `0` (no rate limiting)
+<9> Optional. Replace `<min_tx_rate>` with a Minimum tx rate, in Mbps, for the VF. it should not larger than the `maxTxRate`. The default value is `0` (no rate limiting)
+<10> Optional. Replace `<vlan_qos>` with a IEEE 802.1p Priority levels for the VF. The default value is `0`.
+<11> Optional. Replace `<spoof_check>` with the spoof check mode of the VF. Allowed value are `on` and `off`. The default value is `on`.
```

Comment:
I would suggest small change here, change 'value' to 'values'. So, it would be:

Replace `<spoof_check>` with the spoof check mode of the VF. Allowed values are `on` and `off`. The default value is `on`.

Reply:
maybe worth adding a note here: if `on` or `off` is specified without quotes, then it will be rejected by CRD validation, becasue `on` or `off` will be evaluated as boolean value instead of string, but operator defines this API as string.